### ISSUE TITLE
Cannot use :keyword for dynamic solution inputs

### ISSUE THREAD

**Issue by vektor8891** [zsebtanar/zsebtanar-proto#549]
In PR #541, you cannot use semicolons when defining keys for PLHashMaps because the semicolon will remain in the string after the function is evaluated. E.g:
```
(def solution-fields (const {:prefix "x="})) ;  BAD: this will return ":prefix" after evaluation
```
But you can still use the other way with apostrophes:
```
(def solution-fields (const {"prefix" "x="})) ;  GOOD: this will return "prefix" after evaluation
```

**Comment by vektor8891** [comment]
The problem is that the semicolon is not removed from the key of PLHashMap:
```
(print (keys {:asdf "asdlkjf"})) ;> [":asdf"]
```

**Comment by maxinteger** [comment]
@vektor8891 fixed in pocket-lisp 0.9.0
